apps/site/util/downloadUtils.tsx

```diff
@@ -182,6 +183,14 @@ export const INSTALL_METHODS: Array<
     url: 'https://brew.sh/',
     info: 'layouts.download.codeBox.platformInfo.brew',
   },
+  {
+    label: InstallationMethodLabel.DEVBOX,
+    value: 'DEVBOX',
+    compatibility: { os: ['WIN', 'MAC', 'LINUX'] },
+    iconImage: <PlatformIcons.Devbox width={16} height={16} />,
+    url: 'https://github.com/jetify-com/devbox',
```

Comment:
```suggestion
    url: 'https://www.jetify.com/devbox/',
```

Why not use the official website ?

Reply:
Sure, why not :)